doc/server/hap2/HowToUse.md

```diff
@@ -87,13 +87,32 @@ via pip with the following command:
 
 You need to install hap2 dependent packages via apt-get with the following command:
 
-    $ sudo apt-get install python-pika python-daemon
+    $ sudo apt-get install python-pika
 
 If you want to use hap2-nagios-livestatus, you need to install `python-mk-livestatus`
 via pip with the following command:
 
     $ sudo pip install python-mk-livestatus
 
+
+## How to add HAP2
+
+Each HAP2 plugins RPM put SQL file on /usr/share/hatohol/sql .
+
+You can read the SQL file add monitoring server type by hatohol-db-initistor command.
+
+Please execute the following.
+
+```
+$ hatohol-db-initiator --db-user <YOUR_DB_USER> --db-password <YOUR_DB_PASSWORD>
+```
+
+Success By confirming the following output.
```

Comment:
More concrete descriptions is better.

Please confirm whether or not above command succeeded by the hatohol-db-initiator command output: